.github/workflows/build.yml

```diff
@@ -202,6 +202,10 @@ jobs:
           git config --global user.name "github-actions[bot]"
           git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
 
+      - name: Check for PR merge
+        run: |
+          ./ve1/bin/check-auto-merge --api-url=${{ github.event.pull_request._links.self.href }}
+
```

Comment:
Should not be better if check this right after `merge_pr` step so that we can fail fast? 

Also, any advantages of using pythin here vs. github-script action?